module/VuFind/tests/integration-tests/src/VuFindTest/Mink/RecordActionsTest.php

```diff
@@ -946,6 +946,82 @@ function () use ($page, $checked) {
         $this->clickCss($page, '.logoutOptions a.logout');
     }
 
+    /**
+     * Test adding comments, rating and tag to a record and then
+     * delete them from user account menu
+     *
+     * @return void
+     */
+    public function testUserContentDeletion()
+    {
+        $page = $this->gotoRecord();
+        $this->changeConfigs(
```

Comment:
It might be better to change the configuration before going to the record page, to ensure that the configuration is constant for the entire test.